resources/startup.sh

```diff
@@ -410,6 +467,20 @@ fi
 echo "Removing temporary admin..."
 remove_temporary_admin_user_and_group
 
+# Please note!
```

Comment:
This line is irrelevant, as you could write that on pretty much every comment line

Reply:
Then how would declare such an order problem? It was already unclear in the beginning that's why I ran into import problems.

Reply:
On 2nd reading I understand that you mean not the whole comment block but this specific line and I do agree.

Reply:
The declaration that is coming afterwards is totally fine, I just think it does not need a special headline. Otherwise everyone would declare their problem as notably in the future and the code gets bloated with "THIS IS IMPORTANT!" notes.